src/DssSpell.sol

```diff
@@ -15,24 +15,34 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import "lib/dss-interfaces/src/dss/IlkRegistryAbstract.sol";
-import "lib/dss-interfaces/src/dss/VatAbstract.sol";
+
+interface RwaOutputConduitLike {
+    function kiss(address) external;
+    function diss(address) external;
+}
+
+interface Bumpable {
+    function bump(bytes32, uint256) external;
+}
 
 contract DssSpellAction is DssAction {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/67317d1cacd8cd2ad74b3627680631ec3edc56d6/governance/votes/Executive%20vote%20-%20August%206,%202021.md -q -O - 2>/dev/null)"
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO/governance/votes/Executive%20vote%20-%20August%206,%202021.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2021-08-06 MakerDAO Executive Spell | Hash: 0xe64f6b5cde09dbc711b833a0adba5a2e970fafbeeee848bb7ff5b80e7b54d8d2";
+        "2021-08-20 MakerDAO Executive Spell | Hash: <TODO>";
 
-    // Turn off office hours
-    function officeHours() public override returns (bool) {
-        return false;
-    }
+    // Foundation SC team old deployer address (for removal)
+    address constant SC_DOMAIN_DEPLOYER_07 = 0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf;
+
+    // Genesis broker/dealer address for 6s (for addition):
+    // https://forum.makerdao.com/t/6s-broker-dealer-dai-address/9780
+    address constant GENESIS_6S = 0xE5C35757c296FD19faA2bFF85e66C6B25AC8b978;
```

Comment:
sanity check - is this fine that this EOA never had any activity?

Reply:
This was provided by the representative. @godsflaw and others have seen the address and I can confirm that this is the address that was provided to me in the agreements.